### ISSUE TITLE
providing custom dataset

### ISSUE THREAD

**Issue by arthur-leguennec** [none-master/SAIN#1]
Hello,

I am currently testing your network and I was wondering how the data is organized. How are the inter*.jpg images generated in the code? Because I would like to test SAIN with my data, but I don't understand how to give my data to train or to test. Do I need to generate the interpolation images before ?

Thanks !
Arthur

**Comment by none-master** [comment]
Hi Arthur, 

Sorry for the late reply. The inter*.jpg images corresponded to the stroke-level correspondence info. As stated in the paper, we used SuperGlue to construct this info. 
I have uploaded a file for generating inter*.jpg for your reference. It is not complete and you may need to download SuperGlue and its pre-trained model to run it.

**Comment by arthur-leguennec** [comment]
Hi,

thank you for your response ! Yes, I finally understood that and implemented just few days ago. I'll have a look at your script to better reproduce your results.
So now I'm wondering how do you compute/generate the optical flow ? Did you use an custom model pretrained on an other dataset ? I'm sorry, I don't quite understand this part in your article.
I also noticed what I think is an oversight for data augmentation. You uniformly augment frames and keypoints but don't apply it to optical flows (guide_flo13.npy and guide_flo31.npy I guess). By training the model on my side and applying the increase to the optical flow in addition, my PNSR metric is above the results presented in your article. Not by much (about 21.5) but at epoch 10. I didn't watch the other metric SSIM. 

Thanks !
Arthur

EDIT: by compute the optical flow, I mean how do you compute the region-level correspondence exactly ? Can you share the script ?
